packages/kit/src/views/Staking/pages/ProtocolDetailsV2/index.tsx

```
@@ -92,328 +137,389 @@ function SubscriptionSection({
         </NumberSizeableText>
       </YStack>
     </YStack>
-  );
+  ) : null;
 }
 
 function AlertSection({ alerts }: { alerts: IStakeEarnDetail['alerts'] }) {
+  if (alerts && alerts.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)